src/components/Card/StyledCard.js

```diff
@@ -226,9 +227,9 @@ const StyledCard = ({type, expanded, response}) => {
   )
 }
 StyledCard.propTypes = {
-  type: PropTypes.oneOf(keys(cardTypes)),
+  type: PropTypes.oneOf(['course', 'lesson', 'playlist']),
```

Comment:
Is `keys(cardTypes)` the same as `['course', 'lesson', 'playlist']`? If so, would be better to keep it as `keys(cardTypes)` to use the same source.